CMakeLists.txt

```diff
@@ -92,20 +99,19 @@ option(FMT_OS "Include core requiring OS (Windows/Posix) " ON)
 option(FMT_MODULE "Build a module instead of a traditional library." OFF)
 option(FMT_SYSTEM_HEADERS "Expose headers with marking them as system." OFF)
 
-set(FMT_CAN_MODULE OFF)
-if (CMAKE_CXX_STANDARD GREATER 17 AND (
-  (MSVC AND MSVC_VERSION GREATER 1933) OR
-  ((CMAKE_CXX_COMPILER_ID MATCHES "Clang") AND
-   (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL "16.0"))))
-  set(FMT_CAN_MODULE ON)
+set(FMT_CAN_MODULE ${FMT_MODULE})
+if (CMAKE_CXX_STANDARD GREATER 17)
+  if (MSVC_VERSION GREATER 1933 OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+    set(FMT_CAN_MODULE ON)
+  endif ()
 endif ()
-if (NOT FMT_CAN_MODULE)
+if (FMT_MODULE AND NOT FMT_CAN_MODULE)
   set(FMT_MODULE OFF)
-  message(STATUS "Module support is disabled.")
+  message(WARNING "Module support is disabled!")
 endif ()
 if (FMT_TEST AND FMT_MODULE)
   # The tests require {fmt} to be compiled as traditional library
-  message(STATUS "Testing is incompatible with build mode 'module'.")
+  message(WARNING "FMT_MODULE: Some test are incompatible and disabled!")
 endif ()
```

Comment:
Diese drei Änderungen bringen keine funktionalen Vorteile. Erst Funktion, später Kosmetik. Immerhin muß das muß das Eingang in das originale {fmt} Repository finden und dann will Victor wissen, wozu das gut ist.

Reply:
Solange nicht alle Tests laufen finde ich das wichtig, dann kann das wieder raus.